### ISSUE TITLE
High resolution screens

### ISSUE THREAD

**Issue by yaroslavkulinich** [ayltai/Android-Lib-VerticalMarqueeTextView#4]
On screens with high resolution (Nexus 5), scrolling speed is very slow. 
Please test it.

**Comment by yaroslavkulinich** [comment]
I have solved it in my project.
Problem with Thread.sleep(). On some devices it doesn't work properly.
My code in doInBackground():

```
 long millis = 0 ;
            while (VerticalMarqueeTextView.this.marqueeStarted && !VerticalMarqueeTextView.this.marqueePaused) {

                long curr = System.currentTimeMillis();
                if (curr>=millis){
                    activity.runOnUiThread(runnable);
                    millis = curr + (long) (1000d / VerticalMarqueeTextView.this.marqueeSpeed);
                }
            }
```

**Comment by ayltai** [comment]
Cool thanks for the comment. I have updated the library with your suggestion and published a new version 1.0.1.

In the new version I have remove Thread.sleep() and AsyncTask because, as you notice, Thread.sleep() does not perform well on some devices and on some platforms only one AsyncTask can be run at a time, preventing multiple VerticalMarqueeTextView from animating.
